packages/api/Cargo.toml

```
@@ -23,8 +24,13 @@ sqlx = { version = "0.8.3", features = [
     "bigdecimal",
 ] }
 
+
 models = { workspace = true, features = ["server"] }
 schemars = "0.8.22"
+uuid = "1.15.1"
+rest-api = "0.1.8"
```

use rest-api of rust-sdk submodule instead of crate registry.
Because now, we didn't publish latest version of it.

Refer to Cargo.toml in project root.
you can use as below in Cargo.toml of project root.

```
rest-api = { path = "deps/rust-sdk/packages/rest-api" }
```

Then here
```suggestion
rest-api.workspace = true
```